CAIPs/caip-19.md

```diff
@@ -31,7 +31,7 @@ The Asset Type is a string designed to uniquely identify the types of assets in
 The `asset_type` is a case-sensitive string in the form
 
 ```
-asset_type:    chain_id + "/" + asset_namespace + ":" + asset_reference
+asset_type:    chain_id + ";" + asset_namespace + ":" + asset_reference
```

Comment:
```suggestion
asset_type:    chain_id + "-" + asset_namespace + ":" + asset_reference
```

Would porefer if we use `-` instead because semicolon might break DIDs in the future:
https://www.w3.org/TR/did-core/#did-url-syntax

Reply:
Both CAIP-2 identifiers and asset namespaces may also contain `-`, so with `-` separators CAIP-19 identifiers will be ambiguous. Example:

CAIP-2: `namespace:chain-1`
Asset namespace: `asset-namespace`
Asset reference: `123`

The resulting CAIP-19 asset type would be `namespace:chain-1-token-namespace:123`. It is not clear how it should be parsed.